### PR TITLE
add javascriptreact tag to typescriptreact

### PR DESCRIPTION
**Pull request by uriklar**
looks like a good case for adding this inheritance

### CODE REVIEW

File: lang/typescript/typescriptreact.talon [hunk 1]
```diff
@@ -1,3 +1,4 @@
 tag: user.typescriptreact
 -
 tag(): user.typescript
+tag(): user.javascriptreact
```
Comment (pokey) [inline lang/typescript/typescriptreact.talon:4]: @AndreasArvidsson just wanted to check that there wasn't some reason your original PR didn't include this tag.  Is it a problem that we get a diamond inheritance structure where we activate `javascript` twice?

**Review by pokey — COMMENTED** [review]
This seems reasonable to me but let's wait to see if @AndreasArvidsson had some reason to leave it out originally

File: lang/typescript/typescriptreact.talon (hunk 1 above)
Comment (AndreasArvidsson) [reply lang/typescript/typescriptreact.talon:4]: Nah no problem like that with tags

**Review by pokey — APPROVED** [review]
tested locally; works fine.  Thanks!

**Pull request merged**